### PR TITLE
[mod_login]XML code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_login
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

File: modules/mod_login/mod_login.xml [hunk 1]
```diff
@@ -25,105 +25,115 @@
...
 					rows="5"
-					label="MOD_LOGIN_FIELD_POST_TEXT_LABEL"
-					description="MOD_LOGIN_FIELD_POST_TEXT_DESC" />
+				/>
+
 				<field
 					name="login"
 					type="menuitem"
 					disable="separator,alias,heading,url"
 					label="MOD_LOGIN_FIELD_LOGIN_REDIRECTURL_LABEL"
```
Comment (andrepereiradasilva) [inline modules/mod_login/mod_login.xml]: move disable down?

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 88d4e58c35376fca499aef990b173b7a775d98a6<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10586">issues.joomla.org/joomla-cms/10586</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 88d4e58c35376fca499aef990b173b7a775d98a6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10586">issues.joomla.org/joomla-cms/10586</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10586">issues.joomla.org/joomla-cms/10586</a>.</sub>

**Comment by JoomliC** [comment]
I have tested this item :white_check_mark: successfully on 88d4e58c35376fca499aef990b173b7a775d98a6<br /><br />On code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10586">issues.joomla.org/joomla-cms/10586</a>.</sub>

**Pull request closed without merging**